crates/project-model/src/rustc_cfg.rs

```diff
@@ -2,14 +2,21 @@
 
 use std::process::Command;
 
+use anyhow::Context;
 use rustc_hash::FxHashMap;
 
-use crate::{cfg_flag::CfgFlag, utf8_stdout, ManifestPath};
+use crate::{cfg_flag::CfgFlag, utf8_stdout, ManifestPath, Sysroot};
+
+pub(crate) enum RustcCfgConfig<'a> {
```

Comment:
The name here is a little generic, but I'm struggling to think of anything better. In any case, I hope that the doc comment helps a little.

Reply:
`builtin_cfg` is a good option. While we're bikeshedding, what do you think something like `target_cfg` since the output consists of things like:

<details>
```
rust-analyzer on davidbarsky/use-sysroot-rustc-to-determine-cfgs [$] via 🦀 v1.71.0
❯ rustc --print cfg -O
panic="unwind"
target_arch="aarch64"
target_endian="little"
target_env=""
target_family="unix"
target_feature="aes"
target_feature="crc"
target_feature="dit"
target_feature="dotprod"
target_feature="dpb"
target_feature="dpb2"
target_feature="fcma"
target_feature="fhm"
target_feature="flagm"
target_feature="fp16"
target_feature="frintts"
target_feature="jsconv"
target_feature="lor"
target_feature="lse"
target_feature="neon"
target_feature="paca"
target_feature="pacg"
target_feature="pan"
target_feature="pmuv3"
target_feature="ras"
target_feature="rcpc"
target_feature="rcpc2"
target_feature="rdm"
target_feature="sb"
target_feature="sha2"
target_feature="sha3"
target_feature="ssbs"
target_feature="vh"
target_has_atomic="128"
target_has_atomic="16"
target_has_atomic="32"
target_has_atomic="64"
target_has_atomic="8"
target_has_atomic="ptr"
target_os="macos"
target_pointer_width="64"
target_vendor="apple"
unix
```
</details>